packages/flutter/test/widgets/tap_and_drag_gestures_test.dart

```diff
@@ -359,6 +379,30 @@ void main() {
     expect(events, <String>['down#1', 'dragstart#1', 'dragend#1']);
   });
 
+  testGesture('Beats TapGestureRecognizer when mouse pointer moves past kPrecisePointerPanSlop', (GestureTester tester) {
+    // Regression test for https://github.com/flutter/flutter/issues/122141
+    final TapGestureRecognizer taps = TapGestureRecognizer()
+      ..onTapDown = (TapDownDetails details) {
+        events.add('tapdown');
+      }
+      ..onTapUp =  (TapUpDetails details) {
+        events.add('tapup');
+      }
+      ..onTapCancel = () {
+        events.add('tapscancel');
+      };
+
+    tapAndDrag.addPointer(down6);
+    taps.addPointer(down6);
+    tester.closeArena(6);
+    tester.route(down6);
+    tester.route(move6);
```

Comment:
This move will trigger the mouse drag started but does not cause the tap gesture rejection.

Reply:
Oh I think this is because this commit https://github.com/flutter/flutter/pull/109573/commits/aa2d73b6ea216e5523f784e7f43dc49b94198d21 . I think an internal test was breaking, iirc there was a TextField inside of a Scrollable and it was preventing the Scrollable from scrolling. My solution for this was for the recognizer to wait for all other recognizers before declaring victory. Similar to how `TapGestureRecognizer` does. This may need to be revisited.